src/php/ext/grpc/php_grpc.c

```diff
@@ -249,38 +249,38 @@ void apply_ini_settings(TSRMLS_D) {
   }
 }
 
-static void custom_logger(gpr_log_func_args* args) {
-  TSRMLS_FETCH();
-
-  const char* final_slash;
-  const char* display_file;
-  char* prefix;
-  char* final;
-  gpr_timespec now = gpr_now(GPR_CLOCK_REALTIME);
-
-  final_slash = strrchr(args->file, '/');
-  if (final_slash) {
-    display_file = final_slash + 1;
-  } else {
-    display_file = args->file;
-  }
-
-  FILE *fp = fopen(GRPC_G(log_filename), "ab");
-  if (!fp) {
-    return;
-  }
-
-  gpr_asprintf(&prefix, "%s%" PRId64 ".%09" PRId32 " %s:%d]",
-               gpr_log_severity_string(args->severity), now.tv_sec,
-               now.tv_nsec, display_file, args->line);
-
-  gpr_asprintf(&final, "%-60s %s\n", prefix, args->message);
-
-  fprintf(fp, "%s", final);
-  fclose(fp);
-  gpr_free(prefix);
-  gpr_free(final);
-}
+// static void custom_logger(gpr_log_func_args* args) {
```

Comment:
I would prefer to just remove the code rather than commenting it out.

Also we need to remove this line from the src/php/README: https://github.com/grpc/grpc/blob/master/src/php/README.md?plain=1#L339

Reply:
Fixed.